### PR TITLE
🤖: Update build recipes for SP5

### PR DESCRIPTION
**Pull request by dcermak**
This is an automated pull request to update from branch for-deploy-5

### CODE REVIEW

**Comment by dcermak** [comment]
/vc user=dancermak packages=aspnet-runtime-6.0,dotnet-runtime-6.0,dotnet-6.0 New upstream release 6.0.21

**Comment by dcermak** [comment]
/vc user=dancermak packages=aspnet-runtime-7.0,dotnet-runtime-7.0,dotnet-7.0 New upstream release 7.0.10

File: aspnet-runtime-6.0/Dockerfile [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: bci/dotnet-aspnet:6.0
 #!BuildTag: bci/dotnet-aspnet:6.0-%RELEASE%
-#!BuildTag: bci/dotnet-aspnet:6.0.20
-#!BuildTag: bci/dotnet-aspnet:6.0.20-%RELEASE%
+#!BuildTag: bci/dotnet-aspnet:1.x86_64.rpm
```
Comment (dirkmueller) [inline aspnet-runtime-6.0/Dockerfile:5]: This looks wrong

**Review by dirkmueller — CHANGES_REQUESTED** [review]

File: aspnet-runtime-6.0/Dockerfile (hunk 1 above)
Comment (dcermak) [reply aspnet-runtime-6.0/Dockerfile:5]: Argh, indeed. I have a hunch where this is coming from…

**Comment by dcermak** [comment]
This needs a code fix first

**Pull request closed without merging**